frontend/app_flowy/packages/flowy_editor/example/lib/plugin/text_node_widget.dart

```diff
@@ -0,0 +1,39 @@
+import 'package:flutter/material.dart';
+import 'package:flowy_editor/flowy_editor.dart';
+
+class TextNodeBuilder extends NodeWidgetBuilder {
+  TextNodeBuilder.create({required super.node, required super.renderPlugins})
+      : super.create();
+
+  String get content => node.attributes['content'] as String;
+
+  @override
+  Widget build() {
+    final childrenWidget = buildChildren();
```

Comment:
I prefer children here. Because we are in a 'widget' context.

Reply:
done.